config/db/seed.js

```diff
@@ -315,22 +443,61 @@ async function dropDb() {
 
 async function createParticipations() {
   const users = await User.find();
-  await Participation.create(
-    users.map(user => {
-      challenge: currentChallenge, user;
-    })
-  );
+  users.map(user => {
+    Participation.create({
+      challenge: currentChallenge,
+      user
+    });
+  });
+}
+
+async function createUsers() {
+  const families = await Family.find();
+  const users = require("./json/users.json");
+  users.map(user => {
```

Comment:
you should first map all the users to a family using the random family, like you are trying to do below, except it should be randomly evenly divided. So every 7 users should have a different family. Write that in assignFamilies and call it in here.